api/envoy/config/filter/http/gzip/v2/gzip.proto

```diff
@@ -21,6 +22,9 @@ message Gzip {
   google.protobuf.UInt32Value memory_level = 1 [(validate.rules).uint32 = {gte: 1, lte: 9}];
 
   // Minimum response length, in bytes, which will trigger compression. The default value is 30.
+  // .. attention:
+  //
+  //    **This field is deprecated**. Set the `compressor` field instead.
```

Comment:
Can you mark the various fields that are deprecated with the `deprecated` attribute?